### PR TITLE
Implement RAG scoring logic for quiz

### PR DESCRIPTION
**Pull request by brettminnie**

### CODE REVIEW

File: src/TransformCore/PHE/HayApi/PersistenceBundle/Scoring/Rag/RagScore.php [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    const SCORE_GREEN = 0;
+
+    protected $score = array(
+        'value' => 0,
+        'rag' => ''
+    );
+
+    /**
+     * @param Questionnaire $questionnaire
+     * @return mixed
```
Comment (eddiejaoude) [inline src/TransformCore/PHE/HayApi/PersistenceBundle/Scoring/Rag/RagScore.php]: Mixed?

Comment (brettminnie) [inline src/TransformCore/PHE/HayApi/PersistenceBundle/Scoring/Rag/RagScore.php]: Noticed already, will be fixed in the next commit, should be an array :P

On Tue, Nov 17, 2015 at 1:39 PM, Eddie Abou-Jaoude <notifications@github.com

Comment (eddiejaoude) [inline src/TransformCore/PHE/HayApi/PersistenceBundle/Scoring/Rag/RagScore.php]: :+1:

File: src/TransformCore/PHE/ExporterBundle/Model/Helper/SmokingHelper.php [hunk 2]
```diff
@@ -21,6 +21,7 @@ public static function extractSmoking(Smoking $smoking)
             "smokingFrequency"         => $smoking->getFrequency(),
             "smokingAmount"            => $smoking->getAmount(),
             "smokingCompleted"         => (true === $smoking->isCompleted()) ? "true" : "false",
```
Comment (eddiejaoude) [inline src/TransformCore/PHE/ExporterBundle/Model/Helper/SmokingHelper.php]: true / false, `strings`?

Comment (brettminnie) [inline src/TransformCore/PHE/ExporterBundle/Model/Helper/SmokingHelper.php]: Yes, booleans are not exported correctly to a csv file.

Comment (eddiejaoude) [inline src/TransformCore/PHE/ExporterBundle/Model/Helper/SmokingHelper.php]: ok :smile:

Comment (brettminnie) [inline src/TransformCore/PHE/ExporterBundle/Model/Helper/SmokingHelper.php]: stupid legacy crm support

**Pull request merged**